include/z64camera.h

```diff
@@ -95,18 +95,18 @@
 
 // Camera stateFlags. Variety of generic flags
 #define CAM_STATE_0 (1 << 0) // Must be set for the camera to change settings based on the bg surface
-#define CAM_STATE_1 (1 << 1) // Must be set for Camera_UpdateWater to run
-#define CAM_STATE_2 (1 << 2)
+#define CAM_STATE_CHECK_WATER (1 << 1) // Must be set for Camera_UpdateWater to run
+#define CAM_STATE_2 (1 << 2) //  Must be set for the camera to change settings based on the bg surface
 #define CAM_STATE_3 (1 << 3) // Customizable flag for different functions
-#define CAM_STATE_4 (1 << 4)
-#define CAM_STATE_5 (1 << 5)
-#define CAM_STATE_6 (1 << 6)
+#define CAM_STATE_4 (1 << 4) // Related to respawn mode in player
+#define CAM_STATE_5 (1 << 5) // Prevents camera mode from changing mode depending on flags
```

Comment:
maybe...
```suggestion
#define CAM_STATE_5 (1 << 5) // Prevents camera from changing mode depending on the `flags` passed to `Camera_ChangeModeFlags`
```

also may be worth to note that this flag is unset every frame

Based on context and what it does, we could rename `Camera_ChangeModeFlags`'s `flags` to something like `forceModeChange`

It's true when changing setting, which makes sense imo (otherwise the new setting just wouldn't be applied)

And false for a generic mode change, which then means
- `CAM_STATE_5` may be set by camera functions to lock the mode,
- or if the new mode is the same, the "change" is aborted

and also `CAM_STATE_5` -> `CAM_STATE_LOCK_MODE` idk?

it makes the name for `Camera_ChangeModeFlags` itself weird but I don't have an idea